api/HTMLElement.json

```diff
@@ -27,12 +27,12 @@
             "version_added": "10.1"
           },
           "safari": {
-            "version_added": "1.3"
+            "version_added": "1"
```

Comment:
This change is needed to pass the lint, and is supported by the link given in the description:
https://github.com/WebKit/WebKit/blob/a063c2b75ddebd1edfe9c54c428d762bdb61f794/WebCore/khtml/ecma/kjs_html.cpp